bin/setup_mongo.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```

Comment:
Might want to have a `set -e` here so that failures of individual lines will cause the script to stop.